source/features/clear-pr-merge-commit-message.tsx

```diff
@@ -19,10 +19,12 @@ async function init(): Promise<void | false> {
 	}
 
 	// Preserve closing issues numbers when a PR is merged into a non-default branch since GitHub doesn't close them #4531
-	if (getBranches().base !== await getDefaultBranch()) {
+	if (getBranches().base.branch !== await getDefaultBranch()) {
```

Comment:
On cross-repo PRs, this comparison was between `"user:branch"` and `"branch"`, so it would never match.

This caused https://github.com/refined-github/refined-github/pull/6122 to appear on every cross-repo PR instead of just non-default-branch PRs

Reply:
What about it? This bug was introduced by your commit at [`4a1a31d` (#6122)](https://github.com/refined-github/refined-github/pull/6122/commits/4a1a31dfa1d0738eb28c79a6341d73e0d76b03de), which was prompted by https://github.com/refined-github/refined-github/pull/6122#discussion_r1010161143, which was correct except the incorrect usage.

Reply:
Oh I though you meant something else with your comment.
Never mind